### DISCUSSION TITLE
How to change the category buttons

### DISCUSSION THREAD

**Discussion by Tan-Robotic** [sharadcodes/jekyll-theme-serial-programmer#23] [Q&A / Get Help] [answered]
Hi, I have no experience in web programming. So I want to know how to change the category buttons under each post title. For example, I can link to a website.
![Rock_17555](https://github.com/sharadcodes/jekyll-theme-serial-programmer/assets/142491049/90a75159-5257-4849-8676-866ce6da73b8)
Thanks！！

**Comment by sharadcodes** [comment]
Do you want to remove the category buttons and use custom links or do you want both ?

**Reply by Tan-Robotic** [reply]
both

**Reply by sharadcodes** [reply, marked as answer]
Modify the layout for the blog page also add and array to the markdown files for those links each one will be available then in the loop while you generate the listing
